From 7425e49166e6f16743ec9528697d06bc4103dc5e Mon Sep 17 00:00:00 2001
From: Tim Orling <tim.orling@konsulko.com>
Date: Sun, 20 Feb 2022 18:55:50 -0800
Subject: [PATCH] Change hash bang to python3

In setup.py, smartypants and smartypants.py set hash bang to python3.

Upstream-Status: Inappropriate [oe-core specific]

Signed-off-by: Tim Orling <tim.orling@konsulko.com>

Update to apply on v2.0.2.

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 setup.py       | 2 +-
 smartypants    | 2 +-
 smartypants.py | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index c183fc0..e10d757 100755
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright (c) 2025–present Justin Mayer
 # For detail license information, See COPYING
 
diff --git a/smartypants b/smartypants
index 1dabf7d..20c130e 100755
--- a/smartypants
+++ b/smartypants
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright (c) 2025–present Justin Mayer
 # Licensed under the BSD License, for detailed license information, see COPYING
 
diff --git a/smartypants.py b/smartypants.py
index 62c1472..a440d14 100755
--- a/smartypants.py
+++ b/smartypants.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # Copyright (c) 2025–present Justin Mayer
 # Copyright (c) 2017 Leo Hemsted
 # Copyright (c) 2013, 2014, 2016 Yu-Jie Lin
-- 
2.49.0

